stingray/lightcurve.py

```diff
@@ -70,9 +81,25 @@ def __init__(self, time, counts, input_counts=True):
         if input_counts:
             self.counts = np.asarray(counts)
             self.countrate = self.counts/self.dt
+            if err is not None:
+                assert np.all(np.isfinite(err)), "There are inf or NaN " \
+                                                 "values in your err array!"
```

Comment:
I tried, but the tests failed and `pytest` didn't caught `TypeError` as it should.
I will play with it a little more to see if I find the bug.